gtsam/nonlinear/NonlinearOptimizer.cpp

```diff
@@ -124,41 +126,51 @@ const Values& NonlinearOptimizer::optimizeSafely() {
 }
 
 /* ************************************************************************* */
-VectorValues NonlinearOptimizer::solve(const GaussianFactorGraph& gfg,
-                                       const NonlinearOptimizerParams& params) const {
+VectorValues NonlinearOptimizer::solve(
+    const GaussianFactorGraph& gfg,
+    const NonlinearOptimizerParams& params) const {
   // solution of linear solver is an update to the linearization point
   VectorValues delta;
   boost::optional<const Ordering&> optionalOrdering;
-  if (params.ordering)
-    optionalOrdering.reset(*params.ordering);
+  if (params.ordering) optionalOrdering.reset(*params.ordering);
 
   // Check which solver we are using
   if (params.isMultifrontal()) {
     // Multifrontal QR or Cholesky (decided by params.getEliminationFunction())
     delta = gfg.optimize(optionalOrdering, params.getEliminationFunction());
   } else if (params.isSequential()) {
     // Sequential QR or Cholesky (decided by params.getEliminationFunction())
-    delta = gfg.eliminateSequential(optionalOrdering, params.getEliminationFunction(), boost::none,
-                                    params.orderingType)->optimize();
+    delta = gfg.eliminateSequential(optionalOrdering,
+                                    params.getEliminationFunction(),
+                                    boost::none, params.orderingType)
+                ->optimize();
```

Comment:
Same comment about formatting. Is this recommended?

Reply:
IDK, probably need to ask frank